test/integration/testManualConnection.js

```diff
@@ -8,9 +8,10 @@ const assert = require('assert');
 const connOption = require('./connectionOptions');
 const testUtil = require('./testUtil');
 const Logger = require('../../lib/logger');
+const { laodConnectionConfiguration } = require('../../lib/configuration/connection_configuration');
 
 if (process.env.RUN_MANUAL_TESTS_ONLY === 'true') {
-  describe.only('Run manual tests', function () {
+  describe('Run manual tests', function () {
```

Comment:
why only is not necessary here?

Reply:
We can stay with previous version

Reply:
there are still places with removed `only` - could you check?